test-lit/CMakeLists.txt

```diff
@@ -14,17 +14,12 @@ set(FLANG_TEST_PARAMS
   flang_site_config=${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg.py)
 
 set(FLANG_TEST_DEPENDS
-  flang
   f18
-  llvm-lit
-  FileCheck
-  count
-  not
```

Comment:
We will want to add these back in once #969 lands, but right now these aren't targets that we can depend on because they're not present as targets in our own cmake call.